shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/segment/table/JoinTableAssert.java

```diff
@@ -44,14 +46,21 @@
      */
     public static void assertIs(final SQLCaseAssertContext assertContext, final JoinedTableSegment actual, final ExpectedJoinTable expected) {
         TableFactorAssert.assertIs(assertContext, actual.getTableFactor(), expected.getTableFactor());
-        JoinSpecificationAssert.assertIs(assertContext, actual.getJoinSpecification(), expected.getJoinSpecification());
+        if (null != actual.getJoinSpecification()) {
+            JoinSpecificationAssert.assertIs(assertContext, actual.getJoinSpecification(), expected.getJoinSpecification());
+        }
     }
     
     public static void assertIs(final SQLCaseAssertContext assertContext, final List<JoinedTableSegment> actual, final List<ExpectedJoinTable> expected) {
         assertThat(assertContext.getText("JoinTable size assert error"), actual.size(), is(null == expected ? 0 : expected.size()));
         for (int i = 0; i < actual.size(); i++) {
             TableFactorAssert.assertIs(assertContext, actual.get(i).getTableFactor(), expected.get(i).getTableFactor());
-            JoinSpecificationAssert.assertIs(assertContext, actual.get(i).getJoinSpecification(), expected.get(i).getJoinSpecification());
+            if (null != expected.get(i).getJoinSpecification()) {
+                assertNotNull(assertContext.getText("Actual JoinSpecification segment should exist."), actual.get(i).getJoinSpecification());
+                JoinSpecificationAssert.assertIs(assertContext, actual.get(i).getJoinSpecification(), expected.get(i).getJoinSpecification());
+            } else {
+                assertNull(assertContext.getText("Actual JoinSpecification segment should not exist."), actual.get(i).getJoinSpecification());
```

Comment:
This sentence has the same meaning as line 58, please delete it.

Reply:
Not the same, there is a similar use in file shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/asserts/statement/dml/impl/SelectStatementAssert.java